### ISSUE TITLE
aboutgray image

### ISSUE THREAD

**Issue by Songpenglei123** [ultralytics/yolov5#2346]
How to deal with gray image

**Comment by glenn-jocher** [comment]
@Songpenglei123 no action is needed for grey images, the dataloader converts grey images into 3ch images automatically. COCO dataset has several greyscale images for example.

**Comment by ptran1203** [comment]
For most of CNN models, I stacked the gray image by `np.concatenate([image, image, image], axis=-1)` to create 3 channels image.
It would be beneficial because you did not have to change the network architecture and able to use pre-trained weights

**Comment by Songpenglei123** [comment]
> @Songpenglei123 no action is needed for grey images, the dataloader converts grey images into 3ch images automatically. COCO dataset has several greyscale images for example.

Is the single-channel processing speed faster than the three-channel? I want to retrain specifically for the single-channel. Is it more difficult?

**Comment by Songpenglei123** [comment]
> For most of CNN models, I stacked the gray image by `np.concatenate([image, image, image], axis=-1)` to create 3 channels image.
> It would be beneficial because you did not have to change the network architecture and able to use pre-trained weights

Because the speed of the algorithm is a bit slow now, I wonder if the single channel will be faster

**Comment by glenn-jocher** [comment]
@Songpenglei123 real world speedups from single channel inputs are not material. You can create single channel models for profiling with PyTorch Hub, see PyTorch Hub tutorial:

## Tutorials

* [Train Custom Data](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data)&nbsp; 🚀 RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training)
* [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading)&nbsp; ⭐ NEW
* [ONNX and TorchScript Export](https://docs.ultralytics.com/yolov5/tutorials/model_export)
* [Test-Time Augmentation (TTA)](https://docs.ultralytics.com/yolov5/tutorials/test_time_augmentation)
* [Model Ensembling](https://docs.ultralytics.com/yolov5/tutorials/model_ensembling)
* [Model Pruning/Sparsity](https://docs.ultralytics.com/yolov5/tutorials/model_pruning_and_sparsity)
* [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution)
* [Transfer Learning with Frozen Layers](https://docs.ultralytics.com/yolov5/tutorials/transfer_learning_with_frozen_layers)&nbsp; ⭐ NEW
* [TensorRT Deployment](https://github.com/wang-xinyu/tensorrtx)
